core/src/main/java/org/elasticsearch/common/util/primitives/Integers.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.common.util.primitives;
+
+import java.util.Collection;
+import java.util.Objects;
+
+public class Integers {
+    public static int[] toArray(Collection<Integer> ints) {
+        Objects.requireNonNull(ints);
+        return ints.stream().mapToInt(s -> s).toArray();
+    }
+
+    public static int checkedCast(long value) {
```

Comment:
Nice. Thanks for bringing that method to my attention. I've pushed 12fab4dcc1634d5a7e6b5b9d60b3da504b06e314 to address.